lib/msf/core/modules/metadata/obj.rb

```diff
@@ -96,6 +96,11 @@ def initialize(module_instance, obj_hash = nil)
 
     @notes = module_instance.notes
 
+    if @name =~ /ms08_059_his2006/
```

Comment:
I think this can be removed for this PR.

Reply:
oops